docs/auth.md

```diff
@@ -0,0 +1,149 @@
+# Introduction
+
+kcp implements the RBAC based authorization scheme as Kubernetes.
+Other authorization schemes (i.e. ABAC) are currently not supported.
+Generally, the same (cluster) role and (cluster) role binding priniciples apply the same way as in Kubernetes.
+
+However, since kcp is based around workspaces and workspaces are forming a tenancy boundary,
```

Comment:
sounds little confusing as the org workspace is merely the tenancy boundary. I know what you mean. Maybe just remove this "since ...." sentence and make it a simple statement.

Reply:
agreed, removed